src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -347,4 +347,134 @@
+  </data>
+  <data name="SwitchToTabCommandKey" xml:space="preserve">
+    <value>Switch to tab</value>
+  </data>
+  <data name="NewTabCommandKey" xml:space="preserve">
+    <value>New tab</value>
+  </data>
+  <data name="SplitPaneCommandKey" xml:space="preserve">
+    <value>Split pane</value>
+  </data>
+  <data name="NewWindowCommandKey" xml:space="preserve">
+    <value>New window</value>
+  </data>
+  <data name="DuplicateTabCommandKey" xml:space="preserve">
+    <value>Duplicate tab</value>
+  </data>
+  <data name="DuplicatePaneCommandKey" xml:space="preserve">
+    <value>Duplicate pane</value>
+  </data>
+  <data name="NextTabCommandKey" xml:space="preserve">
+    <value>Next tab</value>
+  </data>
+  <data name="PrevTabCommandKey" xml:space="preserve">
+    <value>Previous tab</value>
+  </data>
+  <data name="ClosePaneCommandKey" xml:space="preserve">
+    <value>Close pane</value>
+  </data>
+  <data name="CloseTabCommandKey" xml:space="preserve">
+    <value>Close tab</value>
+  </data>
+  <data name="SplitHorizontalCommandKey" xml:space="preserve">
+    <value>Split pane horizontally</value>
+  </data>
+  <data name="SplitVerticalCommandKey" xml:space="preserve">
+    <value>Split pane vertically</value>
+  </data>
+  <data name="CopyTextCommandKey" xml:space="preserve">
+    <value>Copy text</value>
+  </data>
+  <data name="CopyTextAsSingleLineCommandKey" xml:space="preserve">
+    <value>Copy text as a single line</value>
+  </data>
+  <data name="PasteTextCommandKey" xml:space="preserve">
+    <value>Paste</value>
+  </data>
+  <data name="ScrollDownCommandKey" xml:space="preserve">
+    <value>Scroll down one line</value>
+  </data>
+  <data name="ScrollDownPageCommandKey" xml:space="preserve">
+    <value>Scroll down one page</value>
+  </data>
+  <data name="ScrollUpCommandKey" xml:space="preserve">
+    <value>Scroll up one line</value>
+  </data>
+  <data name="ScrollUpPageCommandKey" xml:space="preserve">
+    <value>Scroll up one page</value>
+  </data>
+  <data name="AdjustFontSizeCommandKey" xml:space="preserve">
+    <value>Adjust font size</value>
+  </data>
+  <data name="IncreaseFontSizeCommandKey" xml:space="preserve">
+    <value>Increase font size</value>
+  </data>
+  <data name="DecreaseFontSizeCommandKey" xml:space="preserve">
+    <value>Decrease font size</value>
+  </data>
+  <data name="IncreaseFontSizeWithAmountCommandKey" xml:space="preserve">
+    <value>Increase font size, amount: {0}</value>
+    <comment>{0} will be replaced with a positive number</comment>
+  </data>
+  <data name="DecreaseFontSizeWithAmountCommandKey" xml:space="preserve">
+    <value>Decrease font size, amount: {0}</value>
+    <comment>{0} will be replaced with a positive number</comment>
+  </data>
+  <data name="ResetFontSizeCommandKey" xml:space="preserve">
+    <value>Reset font size</value>
+  </data>
+  <data name="ToggleCommandPaletteCommandKey" xml:space="preserve">
+    <value>Toggle command palette</value>

```

Comment:
```suggestion
    <value>Hide command palette</value>
```
You'll only see this via the command palette, right? So, maybe we should just say hide it?